host-bin/mount-chroot

```diff
@@ -64,11 +66,6 @@ promptNewPassphrase() {
     [ -t 0 ] && stty -echo
     while [ -z "$passphrase" ]; do
         read -r passphrase
-        if [ -z "$passphrase" ]; then
```

Comment:
Actually, I think this should still be enforced if keyfile wasn't set by the user.